src/Analysis/Ast/Impl/Specializations/Typing/Types/GenericTypeParameter.cs

```diff
@@ -33,11 +35,21 @@ public GenericTypeParameter(string name, IPythonModule declaringModule, IReadOnl
         public override PythonMemberType MemberType => PythonMemberType.Generic;
         public override bool IsSpecialized => true;
 
-
         public static IPythonType FromTypeVar(IArgumentSet argSet, IPythonModule declaringModule, IndexSpan location = default) {
             var args = argSet.Values<IMember>();
+            var eval = argSet.Eval;
+
             if (args.Count == 0) {
-                // TODO: report that at least one argument is required.
+                var sourceLocation = argSet.CallExpression.GetLocation(eval.Module);
+
+                eval?.ReportDiagnostics(
```

Comment:
we aren't checking `argSet.Eval` for null anywhere in this method.

Reply:
Yup, I'll fix that